### DISCUSSION TITLE
Add Emoji Reactions

### DISCUSSION THREAD

**Discussion by tenwiz** [dailydotdev/daily#917] [Feature Requests]
I was wondering if there is a possibility of adding emoji reactions as an alternative to upvoting. It would be great to have this functionality integrated. Thank you!

**Comment by tenwiz** [comment]
![image](https://github.com/dailydotdev/daily/assets/24917394/5cffe3d1-d3ee-41c2-9529-8d8471af0200)

**Reply by idoshamun** [reply]
Hi @tenwiz,

Thanks for sharing and suggesting 🙏

**Comment by hardikdgsa** [comment]
Is there any update on the same? I am also waiting for the same from last few weeks

**Reply by idoshamun** [reply]
it's not prioritized currently
